src/main/algorithms/search/graphForm/graphFormConverter/visitedItems.ts

```diff
@@ -0,0 +1,41 @@
+import { GraphForm } from '@/main/algorithms/search/graphForm/graphForm'
```

Comment:
graphFormConverter/types.ts is empty => remove or fill it

Reply:
removed